plugins/hls-semantic-tokens-plugin/src/Ide/Plugin/SemanticTokens/Internal.hs

```diff
@@ -68,14 +77,38 @@ computeSemanticTokens :: Recorder (WithPriority SemanticLog) -> PluginId -> IdeS
 computeSemanticTokens recorder pid _ nfp = do
   config <- lift $ useSemanticConfigAction pid
   logWith recorder Debug (LogConfig config)
+  semanticId <- lift getAndIncreaseSemanticTokensId
```

Comment:
possibly extremely dumb suggestion, but... what if we used the request ID for this? I guess maybe JSON-RPC doesn't guarantee that ids are never reused?

Reply:
Or the file version?

Reply:
SemanticTokens depends on the file and its dependencies. We might not find a good identifier except we maintain our own.

Reply:
Yeah, seems reasonable. Maybe write a comment?

Reply:
Added